test/parallel/test-buffer-zero-fill.js

```diff
@@ -3,6 +3,7 @@
 require('../common');
 const assert = require('assert');
 
+// Tests deprecated Buffer API on a purpose
```

Comment:
Maybe write this as "Tests the deprecated Buffer() constructor"

If you don't change it, "on a purpose" should be "on purpose"

Reply:
Technically, the first line of those two (`Buffer(100)`) is not a constructor call, it's the commit description which is a bit inaccurate there (but rewording will make it needlessly longer).

I will fix the language though, thanks!